blockprotocol/blockprotocol
# [BP-2] Set up Þ Storybook scaffolding

🌟 What is the purpose of this PR?
---
This PR sets up a [Storybook instance](https://storybook.js.org/) at `libs/@blockprotocol/design-system`

🔗  Related links
---
- [Github](https://github.com/blockprotocol/blockprotocol/issues/1132)

❓How to test this?
---

1. Go to the directory `cd libs/@blockprotocol/design-system`
2. Start the Storybook instance `yarn storybook`

---
This code was written and reviewed by GitStart Community. Growing future engineers, one PR at a time.


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204114057854782
